### PR TITLE
Checkbox and radio style

### PR DESCRIPTION
**Pull request by anna-wro**
Before:
screen shot 2017-07-04 at 12 52 05

After:
screen shot 2017-07-04 at 12 52 20

### CODE REVIEW

**Comment by kdzwinel** [comment]
There were some UX concerns regarding the new checkboxes - anyone knows what's the status here? Are we going to get new designs soon?

**Comment by anna-wro** [comment]
@aju does it look like a checkbox now? 😉

screen shot 2017-07-13 at 08 19 27

**Comment by burakukula** [comment]
👏

File: src/components/form-elements/checkbox-radio.html [hunk 1]
```diff
@@ -25,9 +25,9 @@ <h3 class="docs-block__header">Checkboxes</h3>
                 <div class="sg-checkbox">
                     <input class="sg-checkbox__element" type="checkbox" id="checkbox-3">
                     <label class="sg-checkbox__ghost" for="checkbox-3">
-                        <svg class="sg-icon sg-icon--adaptive sg-icon--x8">
-                            <use xlink:href="#icon-check"></use>
-                        </svg>
+                      <svg class="sg-icon sg-icon--adaptive sg-icon--x10">
```
Comment (aju) [inline src/components/form-elements/checkbox-radio.html]: So when migrating we will have to change it everywhere :O 
Please make it as major update

Comment (anna-wro) [inline src/components/form-elements/checkbox-radio.html]: ok

**Pull request merged**